src/azul/project/hca/transformers.py

```diff
@@ -168,15 +172,20 @@ def create_documents(self,
                          manifest: List[JSON],
                          metadata_files: Mapping[str, JSON]
                          ) -> Sequence[ElasticSearchDocument]:
+        # Create a bundle object.
         bundle = api.Bundle(uuid=uuid,
                             version=version,
                             manifest=manifest,
                             metadata_files=metadata_files)
+
+        # Create ElasticSearch documents
         for file in bundle.files.values():
             visitor = TransformerVisitor()
+
```

Comment:
It looks like the difference in the blank line is because whitespace was present and is now removed, so the line does not contain unnecessary whitespace? If so, good.

Reply:
👍